Chapter09/recipe-02/cxx-example/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 # project name and language
-project(recipe-04 CXX C Fortran)
+project(recipe-04 LANGUAGES CXX C Fortran)
```

Comment:
Check the name of the recipe here (it should probably be recipe-02)

Reply:
OK I will spawn an issue on it and run a full repo regex sanity check on the naming.